consensus/validator.go

```diff
@@ -152,7 +152,7 @@ func (consensus *Consensus) prepare() {
 
 // sendCommitMessages send out commit messages to leader
 func (consensus *Consensus) sendCommitMessages(blockObj *types.Block) {
-	if consensus.isBackup || blockObj == nil {
+	if consensus.IsBackup() || blockObj == nil {
```

Comment:
can you add an warn log here so we 
1) can confirm the validator node is in backup mode and 
2) know it is in backup mode when troubleshooting missing signature

Reply:
added log